common/updates-model.cpp

```diff
@@ -84,20 +88,29 @@ void updates_model::draw(std::shared_ptr<notifications_model> not_model, ux_wind
             // Draw Firmware update Pane
             // ===========================================================================
             fw_update_needed = draw_firmware_section(not_model, window_name, update, positions, window, error_message);
-
         }
         else 
-        {
-            ImGui::PushFont(window.get_large_font());
+        { // Indicate device disconnected to the user
             ImGui::PushStyleColor(ImGuiCol_Text, white);
-            ImGui::SetCursorPos({ positions.orig_pos.x, positions.y0 - 100 });
             ImGui::SetWindowFontScale(1.5);
-            ImGui::Text("%s","THE DEVICE HAS BEEN DISCONNECTED,");
-            ImGui::SetCursorPos({ positions.orig_pos.x - 100, positions.y0 - 70 });
-            ImGui::Text("%s", "PLEASE RECONNECT IT OR CLOSE THE UPDATES WINDOW.");
-            ImGui::PopFont();
-            ImGui::SetCursorPos({ positions.orig_pos.x + 230, positions.y0 });
+            std::string disconnected_text = "THE DEVICE HAS BEEN DISCONNECTED,";
+            auto disconnected_text_size = ImGui::CalcTextSize(disconnected_text.c_str());
+            auto disconnected_text_x_pixel = positions.w / 2 - disconnected_text_size.x / 2; // Align 2 center
+            auto vertical_padding = 100.f;
+            ImGui::SetCursorPos({ disconnected_text_x_pixel, vertical_padding });
+            ImGui::Text("%s", disconnected_text.c_str());
+
+            std::string reconnect_text = "PLEASE RECONNECT IT OR CLOSE THE UPDATES WINDOW.";
+            auto reconnect_text_size = ImGui::CalcTextSize(reconnect_text.c_str());
+            auto reconnect_text_x_pixel = positions.w / 2 - reconnect_text_size.x / 2; // Align 2 center
+            ImGui::SetCursorPosX(reconnect_text_x_pixel);
```

Comment:
What about the Y? Automatically on the "next" line?

Reply:
Correct, I just do a "carriage return".